### ISSUE TITLE
Delve build error

### ISSUE THREAD

**Issue by Abathargh** [go-delve/delve#2639]
Hi, there is a compilation problem in the latest version of the repository due to an error in the pkg/proc/breakpoints,go, on line 422. This refers to commit 10406f96d539d905fbe8e8d102d7a01e61f26f04

Here is what I did when I got the error:

```bash
go build -o bin ./cmd/dlv
# github.com/go-delve/delve/pkg/proc
pkg\proc\breakpoints.go:422:49: not enough arguments in call to t.Process.BinInfo().Location
        have (reader.Variable, dwarf.Attr, uint64, op.DwarfRegisters)
        want (godwarf.Entry, dwarf.Attr, uint64, op.DwarfRegisters, MemoryReadWriter)
```

This does not happen when using release v1.7.0, which is what I am currently using to debug go code.

As for the usual information:

1. Delve version: Building from master
2. Go version: 1.16.3
3. OS/arch: Windows 10, amd64 - Debian 9 amd64 - Ubuntu 20.10 amd64
4. What did you do? Cloned the repository, attempted to build the project
5. What did you expect to see? A clean build with the dlv.exe binary as an output
6. What did you see instead? a compilation error on build

**Comment by BenjaminPelletier** [comment]
I cannot `go get` this package any more either -- here is the relevant output from my container build:

```
Step 5/28 : RUN go get github.com/go-delve/delve/cmd/dlv
 ---> Running in 880c17e2548c
# github.com/go-delve/delve/pkg/proc
src/github.com/go-delve/delve/pkg/proc/breakpoints.go:422:49: not enough arguments in call to t.Process.BinInfo().Location
	have (reader.Variable, dwarf.Attr, uint64, op.DwarfRegisters)
	want (godwarf.Entry, dwarf.Attr, uint64, op.DwarfRegisters, MemoryReadWriter)
The command '/bin/sh -c go get github.com/go-delve/delve/cmd/dlv' returned a non-zero code: 2
```

**Comment by aarzilli** [comment]
Closing, fixed by f95340a
